db/migrate/20160801203346_change_users_table.rb

```diff
@@ -0,0 +1,36 @@
+# rubocop:disable MethodLength
+class ChangeUsersTable < ActiveRecord::Migration
+  def up
+    remove_column :users, :name
+    add_column :users, :first_name, :string
+    add_column :users, :middle_name, :string
+    add_column :users, :last_name, :string
+    add_column :users, :social_security_number, :string
+    add_column :users, :date_of_birth, :date
+    add_column :users, :phone, :string
+    add_column :users, :street_address, :string
+    add_column :users, :city, :string
+    add_column :users, :state, :string
+    add_column :users, :zip, :string
+
+    add_index :users, :social_security_number
+  end
+
+  def down
+    add_column :users, :name, :string
+
+    remove_index :users, :social_security_number
+
+    remove_column :users, :first_name
+    remove_column :users, :middle_name, :string
+    remove_column :users, :last_name
+    remove_column :users, :social_security_number
+    remove_column :users, :date_of_birth
+    remove_column :users, :phone, :string
+    remove_column :users, :street_address
+    remove_column :users, :city
+    remove_column :users, :state
+    remove_column :users, :zip
+  end
+end
```

Comment:
Rails has a `change` method that can be used instead of both `up` and `down`. http://guides.rubyonrails.org/active_record_migrations.html#using-the-change-method